core/src/main/java/com/graphhopper/routing/VirtualEdgeIterator.java

```diff
@@ -135,6 +132,12 @@ public EdgeIteratorState setName( String name )
     }
 
     @Override
+    public boolean getBoolean(int key, boolean _default, PMap kwargs)
```

Comment:
why the additional PMap here? fetching from a map while traversing millions of nodes should not be favoured. Instead map.get once into a local variable of the Weighting and pass this to this method e.g. via a simple Object param?